src/scaife-widgets/components/TextWidth.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <span
+    :class="['text-width-control', { active }]"
+    @click.prevent="$emit('input', width)"
+    >{{ text }}</span
+  >
+</template>
+
+<script>
+  export default {
+    props: ['value', 'width'],
+    computed: {
+      active() {
+        return this.value === this.width;
+      },
+      text() {
+        return this.width.charAt(0).toUpperCase() + this.width.slice(1);
+      },
+    },
+  };
+</script>
+
+<style lang="scss">
+  @import '../_variables.scss';
```

Comment:
We shouldn't need to import `variables.scss`.  

I think it's better if we use the `style-resources-loader` (or rather document/expect sites that use this package, use it) following the [example in Gridsome docs](https://gridsome.org/docs/assets-css/#global-preprocessor-files-ie-variables-mixins).